build.gradle

```diff
@@ -75,6 +75,7 @@ allprojects {
 
 dependencies {
 	implementation 'com.destroystokyo.paper:paper-api:1.14-R0.1-SNAPSHOT'
+	implementation 'org.spigotmc:spigot-api:1.15-R0.1-SNAPSHOT'
```

Comment:
Since Paper now has 1.15.1 API, shouldn't we be using that instead?

Reply:
I was more testing this. When I put the paper 1.15 api in, it wouldnt build. I dont think it likes the fact of using 2 of the same api/different vers

Reply:
No I mean, removing Spigot's dependency and just adding Paper's

Reply:
if I use just paper 1.15, we get errors due to spigot removing some deprecated API that we are still relying on.

Reply:
Depending on both Spigot and Paper APIs is dangerous, because they have different versions of same classes. If some code was removed, Skript needs to be updated. This has been necessary for almost every large Spigot update since 1.9.

Reply:
I agree with you full. This was more of a placeholder so this could build. I shouldn't have actually committed that change to this PR. 

Ill remove that change so this PR can focus on what it really is focusing on.

Other than that, It seems the only 2 issues to prevent building are due to 2 deprecated spigot methods which were removed.
I would love to fix those up and add them to this PR but it involves reflection (which Im not very good at), and they are nested classes. I tried my best to understand reflection in a nested class but I was not able to make it work.